src/interfaces.jl

```diff
@@ -472,8 +472,7 @@ end
 """
     `plottype(plot_args...)`
 
-The default plot type for any argument is `lines`.
```

Comment:
I don't really understand where this is defined. I though the default was to error.